lib/pavilion/test_config/file_format.py

```diff
@@ -271,6 +271,40 @@ class TestConfigLoader(yc.YamlConfigLoader):
             'test_version', default='0.0',
             help_text="Documented test version."
         ),
+        yc.StrElem(
+            'setup_spack_env', choices = ['True', 'true', 'False', 'false'],
```

Comment:
Don't set choices in here, it interferes with pavilion variable usage (even if we never expect anyone to use variables and such on this value).j

Move this under the 'spack_config' section, and rename it 'enable'.